docs/resources/vmdistributedcluster.md

```diff
@@ -0,0 +1,120 @@
+---
+weight: 12
+title: VMDistributedCluster
+menu:
+  docs:
+    identifier: operator-cr-vmdistributedcluster
+    parent: operator-cr
+    weight: 12
+aliases:
+  - /operator/resources/vmdistributedcluster/
+tags:
+  - CRD
+  - VMDistributedCluster
+---
+
+`VMDistributedCluster` is the Custom Resource Definition for orchestrated update of several VictoriaMetrics clusters. It allows you to define and manage cluster components of a distributed VictoriaMetrics setup and apply changes to them without downtime or disruption.
+
+For a high-level overview of VictoriaMetrics distributed cluster architecture, refer to the official [VictoriaMetrics documentation](https://docs.victoriametrics.com/victoriametrics/cluster-victoriametrics/).
+
+## Specification
+
+The `VMDistributedCluster` resource allows you to configure various aspects of your VictoriaMetrics distributed cluster. The specification includes details about:
+
+*   `paused`: If set to `true`, the operator will not perform any actions on the underlying managed objects. Useful for temporarily halting reconciliation.
+*   `zones`: A list of objects, where each entry either defines or references a `VMCluster` instance.
+*   `vmagent`: A reference to a `VMAgent` object that will be configured to scrape metrics from all `VMCluster` instances managed by this `VMDistributedCluster`.
```

Comment:
`vmagent` here should be described as the global write endpoint in this setup, which replicates data to all zones. It may not perform scraping at all, and only accept remote write requests from other clients.